peachjam_search/views.py

```diff
@@ -710,3 +713,11 @@ def get(self, request, *args, **kwargs):
 
     def has_permission(self):
         return self.request.user.is_authenticated and self.request.user.is_staff
+
+
+class SavedSearchCreateView(CreateAPIView):
+    permission_classes = (IsAuthenticated,)
```

Comment:
Please include the usual django model perms, too.

Reply:
Actually, because we're using htmx for the UI, this probably won't be required, since the forms-powered htmx view will handle this.

Reply:
updated to use the model permissions only